docs/pages/choose-an-edition/teleport-cloud/getting-started.mdx

```diff
@@ -55,10 +55,10 @@ $ docker run -it ubuntu:22.10 /bin/bash
 ```
 
 Your terminal will begin a shell session in the `ubuntu:22.10` container. From
-there, make sure you have installed `curl`:
+there, make sure you have `curl` and `telnet` installed. Both are used by the install script:
```

Comment:
```suggestion
there, make sure you have `curl` and `telnet` installed. When we run a script to install Teleport later in this guide, the script will require both of these tools.
```
Suggested because this is the first time we mention an installation script—"the install script" doesn't refer to anything here.